homework-submission/my app/my-app/src/TodoList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Todos = ({ todos, deleteTodo, addTodo }) => {
```

Comment:
`addTodo` is a not defined and not used property, in other case you may expect to have a value and it may have a place confusion.